### ISSUE TITLE
GPTFF application

### ISSUE THREAD

**Issue by Gengzhenchao** [atomly-materials-research-lab/GPTFF#9]
Dear gptff Developers，
        I would like to ask if it is feasible to use GPTFF to study the wettability between two inorganic materials?I am very much looking forward to your reply.

**Comment by luke-24166** [comment]
Yes, if sufficient data is available, you can either fine-tune an GPTFF or train one from scratch, which will possess accurate domain knowledge specific to your modeling system.
